sbt-bloop/src/main/scala/bloop/SbtBloop.scala

```diff
@@ -127,6 +130,9 @@ object PluginImplementation {
       // We cannot depend on `managedSources` and `managedResources` because they trigger compilation
       (Keys.sourceGenerators.value ++ Keys.resourceGenerators.value).join.map(_.flatten)
 
+      // Copy the resources, so that they're available when running and testing
+      val _ = Keys.copyResources.value
```

Comment:
Do you know if `copyResources` triggers generators so that we can remove the previous line?

Reply:
It does, I'll remove it.